### DISCUSSION TITLE
Hide the  Hint Message on screen

### DISCUSSION THREAD

**Discussion by Yihen-Liu** [mrcjkb/rustaceanvim#471] [Q&A]
### Feature description

![image](https://github.com/user-attachments/assets/d731402f-b0d7-4e14-9178-73b1f774b2ce)

When  I setup the rust project, there are lots of Hint message display on screen, it is chaos， how to hide them by setting params?

**Comment by mrcjkb** [comment]
Hey 👋 

You can fine-tune which diagnostics to enable [in the rust-analyzer settings](https://rust-analyzer.github.io/manual.html#diagnostics).

**Reply by Yihen-Liu** [reply]
I set  "rust-analyzer.diagnostics.disabled" in my local macbook, ~/.config/nvim/coc-settings.json as follows:
{
"rust-analyzer.diagnostics.disabled": ["inactive-code", "unreachable-label"]
}

It doesn`t work. the Hint Message still displays.

**Reply by mrcjkb** [reply]
coc-settings.json is is for coc.nvim, not rustaceanvim. 

https://github.com/mrcjkb/rustaceanvim?tab=readme-ov-file#gear-advanced-configuration
